tests/fixed/example-class.php

```diff
@@ -25,20 +25,10 @@ class Example implements IteratorAggregate
 {
     private const VERSION = PHP_VERSION - (PHP_MINOR_VERSION * 100) - PHP_PATCH_VERSION;
 
-    private int|null $foo = null;
-
-    /** @var string[] */
```

Comment:
This comment is lost. Should I report a bug about it? I've tried adding an `@param` phpdoc to the constructor, and it stays.

In your projects, when you use CPP, do you use `@param`, or do you annotate properties, like this:

```php
public function __construct(
    /** @var list<string> */
    private array $someProperty,
   …
) {
```

Reply:
I use the following:
```php
/** @param list<EntitySlot> $slots */
public function __construct(
    private readonly array $slots,
) {
}
```

Reply:
Same as @morozov

Reply:
I think, this is fine. We have other sniffs plus static analysis in place that should already complain about the missing `@param` annotation.

Reply:
There seems to be an agreement. What should I do then? Use `@param` in `tests/fixed/example-class.php` and fix the patches so that they apply cleanly?

Reply:
> Or just keep as is so that this disappearance stays documented?

Keeping it as it is, is fine I guess.